subtests/docker_cli/kill_stopped/kill_utils.py

```diff
@@ -336,7 +336,7 @@ def run_once(self):
         _check = self.config['check_stdout']
         timeout = self.config['stress_cmd_timeout']
         self.sub_stuff['kill_results'] = []
-        stopped_log = False
```

Comment:
R0204:350,20: kill_check_base.run_once: Redefinition of stopped_log type from bool to set